cli/batch.js

```diff
@@ -122,7 +123,7 @@ ${getGraph(issue, '* ', showFullManifest)}
 
 ${issue.issueData.description}
 - [${issue.id}](${issue.issueData.url})
-</details>`
+${getIdentifiers(issue)}</details>`
```

Comment:
I noticed that the content rendered inside of `<details>` isn't searchable (e.g. I cannot find https://github.com/elastic/security/issues/482 by `"CVE-2021-3749"`), do you think we can move identifiers outside of `<details>` or make them searchable in any other way?

Reply:
Oh! That's a bummer. I didn't even think of that. I'll find a good workaround...

Reply:
Option no 1 - Include IDs in header:

<img width="811" alt="image" src="https://user-images.githubusercontent.com/10602/161475633-8acb7b5e-e9e4-447d-8cb0-d2bc304033bd.png">

Option no 2 - Comma separated list of IDs at the bottom of body:

<img width="827" alt="image" src="https://user-images.githubusercontent.com/10602/161475930-958ad1f3-ef28-45ea-8179-b8f23cb9ca5a.png">

Reply:
I've updated the PR to implement option number one (just as an example, we can change it to something else if you like).

As you might have noticed, we only use the `<details>` block when in batch mode. So, I've kept the IDs at the bottom of the body when not in batch mode and included them in the title when in batch mode.

Reply:
Btw, I've verified that anything inside of the `<summary>` block is searchable even though it's technically inside the `<details>` block as well.

Reply:
> I've updated the PR to implement option number one (just as an example, we can change it to something else if you like).

Sounds good to me, thanks for the change!

> Btw, I've verified that anything inside of the summary block is searchable even though it's technically inside the details block as well.

Ah, good to know